yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -1524,39 +1560,183 @@ export class Archiver extends (EventEmitter as new () => ArchiverEmitter) implem
   }
 
   public async rollbackTo(targetL2BlockNumber: BlockNumber): Promise<void> {
+    // TODO: This still assumes 1 block per checkpoint
```

Comment:
```suggestion
    // TODO(pw/mbps): This still assumes 1 block per checkpoint
```
Or something so we can search for it later!